packages/devtools_app/lib/src/screens/profiler/cpu_profiler.dart

```diff
@@ -39,11 +39,9 @@ class CpuProfiler extends StatefulWidget {
         tabs = [
           if (summaryView != null)
             _buildTab(key: summaryTab, tabName: 'Summary'),
-          if (!data.isEmpty) ...[
-            _buildTab(key: bottomUpTab, tabName: 'Bottom Up'),
-            _buildTab(key: callTreeTab, tabName: 'Call Tree'),
-            _buildTab(key: flameChartTab, tabName: 'CPU Flame Chart'),
-          ],
+          _buildTab(key: bottomUpTab, tabName: 'Bottom Up'),
```

Comment:
Can the tabs handle no data now?

Reply:
Yeah. This is necessary because if a filter is applied that results in no stack frames, the user needs to still have a way to widen the filter. When we got rid of the profiler for an "empty" profile, there was no way for the user to get back to the data they were looking at and filtering.